### DISCUSSION TITLE
Convert page.get_drawings() lines to rect

### DISCUSSION THREAD

**Discussion by maimaik05** [pymupdf/PyMuPDF#2513] [Looking for help]
I have a pdf that I am trying to extract the coordinates of the outline out of, to get the coordinates of the outer rectangle if possible. When I use page.get_drawings(), I mostly get path['items'] that are lines, not rects or quads, which doesn't exactly help with what I need. The shapes are not exact rectangles and squares (imagine the outlines of rooms on a floorplan) so I'm wondering if this is even possible? Even just a rectangle overlapping the outlines would do. Unfortunately I'm not able to provide an example, so if you need more information please let me know. Thank you!

**Comment by JorjMcKie** [comment]
The dictionaries returned by `page.get_drawings()` also have a "rect" key, which is the smallest rectangle containing all the `path["items"]`.
Maybe this is useful?

**Reply by maimaik05** [reply]
I think this will help, thank you! I will let you know if anything else comes up
